### PR TITLE
Evaluator order

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Please see the commit messages for details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1246)

### CODE REVIEW

File: cli/src/main/kotlin/commands/EvaluatorCommand.kt [hunk 1]
```diff
@@ -97,7 +97,7 @@ object EvaluatorCommand : CommandWithHelp() {
             return if (evaluator.checkSyntax(script)) 0 else 2
         }
 
-        val evaluatorRun = evaluator.run(script)
+        val evaluatorRun by lazy { evaluator.run(script) }
```
Comment (sschuberth) [inline cli/src/main/kotlin/commands/EvaluatorCommand.kt]: Commit message nits:
- typo in "Delay" (?)
- I do not really get the body text, s/is/if/?

File: .appveyor.yml [hunk 2]
```diff
@@ -53,9 +53,9 @@ build_script:
 
 test_script:
   - if "%APPVEYOR_SCHEDULED_BUILD%"=="True" (
-      gradlew --stacktrace -Pkotlin.parallel.tasks.in.project=false dokkaJar check
+      gradlew --stacktrace dokkaJar check
```
Comment (sschuberth) [inline .appveyor.yml]: Commit message: Could you please add a link to https://youtrack.jetbrains.com/issue/KT-29525?

File: cli/src/main/kotlin/commands/EvaluatorCommand.kt (hunk 1 above)
Comment (mnonnenmacher) [reply cli/src/main/kotlin/commands/EvaluatorCommand.kt]: Improved the commit message.

File: .appveyor.yml (hunk 2 above)
Comment (mnonnenmacher) [reply .appveyor.yml]: Done.

File: cli/src/main/kotlin/commands/EvaluatorCommand.kt (hunk 1 above)
Comment (sschuberth) [reply cli/src/main/kotlin/commands/EvaluatorCommand.kt]: Yet another nit: "*the* check"

Comment (mnonnenmacher) [reply cli/src/main/kotlin/commands/EvaluatorCommand.kt]: Done.

**Pull request merged**